.gitignore

```diff
@@ -0,0 +1,33 @@
+# Ignore .idea
+.idea/
+
+# Ignore configuration files that may contain sensitive information.
+sites/example.sites.php
+docroot/sites/default/.settings.php.swp
+docroot/sites/default/settings.local.php
+
+# Ignore paths that contain generated content.
+files/
+sites/*/files
+sites/*/private
+sites/*/translations
+
+# Ignore default text files
+robots.txt
+/CHANGELOG.txt
+/COPYRIGHT.txt
+/INSTALL*.txt
+/LICENSE.txt
+/MAINTAINERS.txt
+/UPGRADE.txt
+/README.txt
+sites/README.txt
+sites/all/libraries/README.txt
+sites/all/modules/README.txt
+sites/all/themes/README.txt
+
+#Ignore vendor drupal
+docroot/vendor/drupal/
```

Comment:
why is vendor/drupal being ignored?

Reply:
I just thought it should be. I can add them back in.

Reply:
right now, the vendor folder is important. later, it won't be.